src/services/Service.js

```diff
@@ -81,18 +81,26 @@ module.exports = (Database, BaseRelation, Logger, Env, Model) => {
      * @param {Model} param.model Model instance
      * @param {ServiceContext} param.trx Knex transaction
      */
-    async actionCreate ({ model, serviceContext = {} }) {
-      const { error } = await model.validate()
-
-      if (error) {
-        return new ServiceResponse({ error })
+    async actionCreate ({ model, modelData, serviceContext = {} }) {
```

Comment:
I dont think thats a good approach since e already have this logic separated in two actions, merge the utilization of one in another has no improvement at all, since in one way or another the logic for using these functions should be separated in any way

Reply:
@engylemure I've recommended using a single method for both cases: passing a model or a literal object. I think this way we have a more flexible and intuitive interface, since it leaves the job of instantiating the model for the service and not requiring the developer to decide which method to use.